src/main/java/tigase/vhosts/VHostJDBCRepository.java

```diff
@@ -157,10 +159,9 @@ public void addItemNoStore(VHostItem item) {
 		}
 		super.addItemNoStore(item);
 		if (VHostItem.DEF_VHOST_KEY.equals(item.getKey())) {
-			this.defaults = defaults;
```

Comment:
Well, it wasn't updated in above code (`this.defaults = defaults;`) either as it assigned same value to itself. This could have been an error and correct one would be witch currently loaded `item` (which is default, and which would made sense):
```java
this.defaults = item;
```